### ISSUE TITLE
Support hot reload in layout x-slot blocks?

### ISSUE THREAD

**Issue by gsxdsm** [defstudio/vite-livewire-plugin#132]
When I have a livewire full page component and I reference named slots, changes to the contents of the slot don't get hot reloaded properly.

**Comment by fabio-ivona** [comment]
Hi, could you set up a simple project in order to recreate your issue?

**Comment by gsxdsm** [comment]
Here's an example project: 
https://github.com/gsxdsm/livewire-vite-test

See: https://github.com/gsxdsm/livewire-vite-test/blob/main/resources/views/livewire/test.blade.php

When the main contents are updated, the hot reload works correctly. However when the header slot is updated, it does not hot reload (I guess because that's technically not a livewire component?)

**Comment by fabio-ivona** [comment]
yep, I think it is due to the fact that it is not a Livewire component 

we'll investigate on that
